.github/workflows/deploy.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     with:
       github_environment: uat
       environment_name: uat
-      gpfd_url: laa-get-payments-finance-data-api-uat.apps.live.cloud-platform.service.justice.gov.uk
+      gpfd_url: -laa-get-payments-finance-data.cloud-platform.service.justice.gov.uk
```

Comment:
Should it be 'uat-laa', or should it be generic? If it's meant to be generic, you might need to remove the hyphen.